src/chronometer.js

```diff
@@ -1,33 +1,59 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
+  this.currentTime=0;
+  this.intervalId=null;
   }
-
+/////////////////////////////////////////////////////////////////
   start(printTimeCallback) {
-    // ... your code goes here
-  }
 
-  getMinutes() {
-    // ... your code goes here
+let contadorM=0;
+let contadorS=0;
+setInterval(()=> {
+   this.currentTime++
+    if(contadorS<3){
+    }  if( contadorM==0){
+
+    }
+   }, 1000);
   }
+/////////////////////////////////////////////////////////////
+getMinutes() {
+
+var resul=this.currentTime/60;
+var result=Math.floor(resul);
+return result;
+}
 
   getSeconds() {
-    // ... your code goes here
+   var result=this.currentTime%60;
+   return result;
+  
   }
 
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    var str = value.toString(); 
+    if(value<10){
+      return 0+str ;
+  }else{
+      return str;
   }
 
-  stop() {
-    // ... your code goes here
   }
 
+  stop() {
+        clearInterval(Chronometer)
```

Comment:
Se le tiene que asignar el intervalo a `this.intervalId` cuando empiezas tu cronómetro,  para poder almacenar el Id y luego detenerlo en la función stop( ).